active/fundraising.md

```diff
@@ -2,31 +2,60 @@
 
 ## Scope of responsibilities
 
-Coordinates fundraising efforts, particularly around corporate and major donations.
+This working group coordinates fundraising efforts, particularly around corporate and major donations.
+
+The goals of the group are:
+
+- Acquire funding to secure the financial future of Django and support future growth of Django
+- Build and strengthen relationships between Django and donor sources
+- Raise awareness of the Django Web Framework and its community with the broader world
+
+Delegated responsibilities:
+
+- Contacting, developing and implementing financial sponsorships with donor sources
+- Clarifying and elaborating on the various creative strategies that donor sources of different sizes and types can use to support Django
 
-Delegated responsibilities TBD.
 
 ## Initial membership
 
 - Chair: TBD
 - Co-Chair: TBD
 - Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
-- Other members: TBD
+- Other members:
+  - Jacob Kaplan-Moss
+  - Priya Pahwa
+  - Thibaud Colas
+  - Tom Carrick
+  - Sarah Abderemane
+  - Stephanie Goulet
+
 
 ## Future membership
 
 Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
 
-This committee is currently forming and seeking volinteers, including chair and co-chair. 
+This committee is currently forming and seeking volunteers, including chair and co-chair. 
+
+### Navigating Conflicts of Interest
+Each new member must disclose any potential conflicts of interest. These disclosures will be made transparent by means of ____. 
+
+Any person with a conflict of interest will need to recuse themselves from any votes or other decision-making related to their conflict.
+
+No more than X% of the Fundraising Working Group may contain individuals associated with the same organization
```

Comment:
A revision of the CIO section based on the above two comment threads, my opinion, and reading the DSF COI policy:
```suggestion
Members must disclose possible conflicts of interest (COI) to the Fundraising Working Group when they first self-nominate as well as if new COIs arise. No more than two members of the Fundraising Working Group may receive compensation from the same organization at any given time. 

In the spirit of aligning with the [Django Software Foundation Conflict of Interest Policy](https://www.djangoproject.com/foundation/conflict-of-interest/), members must disclose possible COIs that might affect a vote before that vote. That member can then choose to either recuse themselves from the vote or disclose all relevant facts regarding the COI. If they disclose all relevant facts, the remaining members will majority vote whether it will constitute a COI without them present. Ties will be brought to the board to decide prior to vote. The resolution of each COI must be recorded in the Fundraising Working Group notes.
```
My main question is: **do members of the Fundraising Working Group legally counted under the definition of an "Interested Person" from the official DSF COI Policy?** If they do, this section should explicitly state that we defer to that policy (and should also probably be simplified to mainly just say something like "Operates under the DSF COI Policy" with a link to that policy page).

Reply:
Well, first time I hear of this policy! That’s certainly handy to have that all written down 😄 Yes, if this policy is under active use, to me it’s quite clear that a member of a working group is a "member of a committee with governing board delegated powers".

And with that in mind, I’d prefer we defer to that policy without any re-interpretation.